openmdao/components/indep_var_comp.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, name, val=None, **kwargs):
                              "`str` or an iterable of tuples of the form (name, value) or "
                              "(name, value, keyword_dict).")
 
-    def apply_linear(self, mode, ls_inputs=None, vois=(None, ), gs_outputs=None):
+    def sys_apply_linear(self, mode, ls_inputs=None, vois=(None, ), gs_outputs=None):
```

Comment:
should these functions be underscored (non-user facing) and if so, maybe the "sys" prefix is not needed?